README.md

```diff
@@ -407,6 +409,8 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Monitor and limit your docker image size](https://github.com/wemake-services/docker-image-size-limit)
 - [Publish Docker Images to the Amazon Elastic Container Registry (ECR)](https://github.com/appleboy/docker-ecr-action)
 - [Build And Push Your Docker Images Caching Each Stage To Reduce Build Time](https://github.com/whoan/docker-build-with-cache-action)
+- [Update a container repository description from README.md (supported registries: Docker Hub, Quay, Harbor)](https://github.com/marketplace/actions/update-container-description-action)
```

Comment:
Can you please title case this as per CONTRIBUTING.md, and move the supported registries to a description instead of within the link? Thanks

Reply:
done